WcaOnRails/app/models/wcif.rb

```diff
@@ -13,6 +13,23 @@ def json
     {
       "formatVersion" => "1.0",
       "id" => competition.id,
+      "events" => competition.competition_events.map(&method(:competition_event_to_json)),
```

Comment:
Haha. That is actually how I first wrote it, but figured there might be a better way, so I searched for "ruby convert method to a block", and landed on the `&method` solution. It sounds like this isn't a common idiom in ruby, so I prefer your, more obvious implementation.

However, I've updated this PR to us a more OO flavored approach, let me know what you think.